src/components/action-popover/action-popover-item.component.js

```diff
@@ -88,17 +92,22 @@ const MenuItem = ({
 
   const onClick = useCallback(
     (e) => {
-      if (!disabled) {
+      if (!disabled && !isHref && !download) {
         onClickProp();
         setOpenPopover(false);
         focusButton();
         e.stopPropagation();
+      } else if (!disabled && isHref && download) {
+        setOpenPopover(false);
+        focusButton();
+        e.stopPropagation();
       } else {
         ref.current.focus();
+        e.preventDefault();
         e.stopPropagation();
```

Comment:
Saying as every part of this if statement has `e.stopPropagation` you can just move it outside of the `if` and only have it once